defaults/main.yml

```diff
@@ -1,13 +1,19 @@
 ---
+platform_client_deploy_flavor: pull-and-build
```

Comment:
I kinda wonder if it'd just be easier to check for `platform_client_deploy_src` rather than having the extra switch?

Either way - we should add a doc comment above these vars